### DISCUSSION TITLE
Wallbox Without WIFI

### DISCUSSION THREAD

**Discussion by kujrv1** [jagheterfredrik/wallbox-mqtt-bridge#56] [Q&A]
I have a Wallbox pulsar only with Bluethooh. How can i integrate with HA

**Comment by lostOzone** [comment]
You need an ESP32 with https://github.com/jagheterfredrik/wallbox-ble

**Comment by kujrv1** [comment]
Sorry but its not pairing with wallbox. I open one issue in wallbox-ble with the information
Enviado desde mi IPhone

El 27 abr 2024, a las 12:54, lostOzone ***@***.***> escribió:

﻿

You need an ESP32 with https://github.com/jagheterfredrik/wallbox-ble
